app/decorators/campaign_decorator.rb

```diff
@@ -0,0 +1,17 @@
+class CampaignDecorator < ApplicationDecorator
+  include ActionView::Helpers::AssetTagHelper
```

Comment:
I included these as the most convenient way to get access to `image_tag` and `link_to`. We can probably be smarter about this if it bothers people.